docs/examples/django/README.rst

```diff
@@ -100,6 +99,13 @@ output similar to this one:
 The last output shows spans automatically generated by the OpenTelemetry Django
 Instrumentation package.
 
+Disabling Django Instrumentation
+--------------------------------
+
+Django's instrumentation can be disabled by setting the following environment variable.
+
+#. ``export OTEL_PYTHON_DJANGO_INSTRUMENT=False``
```

Comment:
I like [`.. envvar::` directive](https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#directive-envvar) to keep track of all the envvars we have. Then they show up in the doc index nicely. Maybe in the module's docblock and then reference it here with `` :envvar:`OTEL_PYTHON_DJANGO_INSTRUMENT` ``?

Related https://github.com/open-telemetry/opentelemetry-python/issues/1147

Reply:
Agree but probably better if handled in a separate PR. One PR per package would provide a nice incremental improvement.